socorro/processor/breakpad_transform_rules.py

```diff
@@ -374,6 +380,22 @@ def _execute_external_process(self, command_line, processor_meta):
         return external_command_output, return_code
 
     #--------------------------------------------------------------------------
+    @staticmethod
+    def dot_save(a_mapping, key, value):
```

Comment:
@peterbe I looked around and didn't see an already-existing utility method... I am inclined to leave this alone unless you have a more specific suggestion :)